test/test_torch.py

```diff
@@ -7021,8 +7021,6 @@ def tmp(dtype, device):
 # - self_position (=-1), the position of self in the arg list, -1 means skip function check
 # - test_out (=False), whether to test the out= version of the operator
 tensor_op_tests = [
-    ('add', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1e-2),
```

Comment:
These add, addmm, and addmv updates probably belong in other PRs, right?

Reply:
I also removed them here in order to prevent potential merge conflicts. Should we remove these entries after all the OpInfos would've been added, so that they're not potentially reintroduced?

Reply:
Yep; we should remove these as we add OpInfos since the tests are redundant. Eventually we'll get rid of these other test generators completely (like we're getting rid of method_tests()).

Reply:
Entries from `test_torch.py` & `test_autograd.py` should probably be removed when all `method_tests` would have been ported to OpInfos, so that there won't be any merge conflicts, such as [this one](https://github.com/pytorch/pytorch/pull/55915/commits/76884b90c0de9149fa09860e47e441eabfdbc5cf).